app/uk/gov/hmrc/directdebitupdateemailbackend/controllers/UpdateEmailVerificationResultController.scala

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.directdebitupdateemailbackend.controllers
+
+import cats.syntax.eq._
+import com.google.inject.Inject
+import ddUpdateEmail.crypto.CryptoFormat.OperationalCryptoFormat
+import ddUpdateEmail.models.EmailVerificationResult
+import ddUpdateEmail.models.journey.{Journey, JourneyId}
+import ddUpdateEmail.utils.Errors
+import io.scalaland.chimney.dsl.TransformerOps
+import play.api.mvc.{Action, ControllerComponents}
+import uk.gov.hmrc.directdebitupdateemailbackend.actions.Actions
+import uk.gov.hmrc.directdebitupdateemailbackend.services.JourneyService
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class UpdateEmailVerificationResultController @Inject() (
+    actions:        Actions,
+    journeyService: JourneyService,
+    cc:             ControllerComponents
+)(implicit
+    ec: ExecutionContext,
+  cryptoFormat: OperationalCryptoFormat
+) extends BackendController(cc) {
+
+  def updateStartVerificationJourneyResult(journeyId: JourneyId): Action[EmailVerificationResult] =
+    actions.authenticatedAction.async(parse.json[EmailVerificationResult]){ implicit request =>
+      for {
+        journey <- journeyService.get(journeyId)
```

Comment:
Don't know if it's worth adding in a JourneyRequest sort of thing so we can move `journey <- journeyService.get(journeyId)` out of the controllers, like we have in essttp? Not fussed though

Reply:
It's not exactly masses of duplication, only the one line lol

Reply:
👍 overall it'd probably be actually more code to take it out in a refiner. Cos it's just one line I'm tempted to say leave it for now perhaps